java-runtime-builder-app/src/main/java/com/google/cloud/runtimes/builder/buildsteps/MavenBuildStep.java

```diff
@@ -53,7 +53,7 @@ protected void doBuild(Path directory, Map<String, String> metadata) throws Buil
 
   @VisibleForTesting
   String getMavenHome() {
-    return System.getenv("M2_HOME");
+    return System.getenv("MAVEN_HOME");
```

Comment:
Why the change? I believe `M2_HOME` is the convention since Maven 2. Not important though.